packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.stories.ts

```diff
@@ -99,8 +99,6 @@ const formatDescription = `Configures the way that the numeric value is formatte
         </li>
         <li>\`decimal\`: Values as are formatted as decimal values, always displaying \`decimal-digits\` digits after the separator and never displaying exponential notation.
```

Comment:
Let's add a sentence to the `decimal-digits` documentation mentioning that you can set that attribute to 0 to display integers without a decimal separator.